build/stf-run-ci/tasks/setup_stf_local_build.yml

```diff
@@ -55,7 +57,7 @@
   ansible.builtin.shell:
     chdir: "{{ base_dir }}"
     cmd: |
-      LOGFILE="{{ ansible_env.HOME }}/sto_gen_bundle.log" \
```

Comment:
Since the ansible_env is being removed, it's likely that the "gather_facts:" in build/run-ci.yaml can also be removed.